opm/simulators/wells/MultisegmentWellSegments.hpp

```diff
@@ -183,6 +186,41 @@ class MultisegmentWellSegments
     Scalar mixtureDensity(const int seg) const;
     Scalar mixtureDensityWithExponents(const int seg) const;
     Scalar mixtureDensityWithExponents(const AutoICD& aicd, const int seg) const;
+
+    // this class is used to store the result of phase property calculation
+    struct PhaseCalcResult {
+        explicit PhaseCalcResult(const std::size_t num_quantities)
+            : b(num_quantities, 0.0)
+            , mix(num_quantities, 0.0)
+            , mix_s(num_quantities, 0.0)
+            , phase_viscosities(num_quantities, 0.0)
+            , phase_densities(num_quantities, 0.0)
+        {}
+
+        void clear() {
+            std::fill(b.begin(), b.end(), 0.0);
+            std::fill(mix.begin(), mix.end(), 0.0);
+            std::fill(mix_s.begin(), mix_s.end(), 0.0);
+            std::fill(phase_viscosities.begin(), phase_viscosities.end(), 0.0);
+            std::fill(phase_densities.begin(), phase_densities.end(), 0.0);
```

Comment:
Sorry, missed this before.  Since these `std::fill()` calls are now in the `MultisegmentWellSegments.hpp` header, we should include `<algorithm>` here to guarantee that we have a declaration of this function template in scope.  Otherwise, we're depending on other headers including `<algorithm>` for us and that's a potential build break in the future.